Scroller/tests/EditableWrapper-specs.js

```diff
@@ -0,0 +1,123 @@
+import SpotlightContainerDecorator from '@enact/spotlight/SpotlightContainerDecorator';
+import '@testing-library/jest-dom';
+import {render, fireEvent, screen} from '@testing-library/react';
+
+import Button from '../../Button';
+import IconItem from '../../IconItem';
+import Scroller from '../Scroller';
+
+const keyDownUp = (keyCode) => (elm) => {
+	fireEvent.keyDown(elm, {keyCode});
+	return fireEvent.keyUp(elm, {keyCode});
+};
+
+const pressEnterKey = keyDownUp(13);
+
+const populateItems = ({index}) => {
+	const iconItemProps = {
+		background: '#ffffff',
+		bordered: true,
+		icon: '',
+		image: null,
+		label: null,
+		labelColor: null,
```

Comment:
I think null is not proper value for labelColor. Could you please check iconItemProps?

Reply:
I changed iconItemProps values. Please check.